client/lib/plans/plans-list.js

```diff
@@ -438,7 +456,14 @@ const getPlanJetpackSecurityDailyDetails = () => ( {
 const getPlanJetpackSecurityRealtimeDetails = () => ( {
 	group: constants.GROUP_JETPACK,
 	type: constants.TYPE_SECURITY_REALTIME,
-	getTitle: () => translate( 'Security {{em}}Real-time{{/em}}', { components: { em: <em /> } } ),
+	getTitle: () =>
+		getJetpackCROActiveVersion() === 'v2'
+			? translate( 'Jetpack Security {{em}}Real-time{{/em}}', { components: { em: <em /> } } )
+			: translate( 'Security {{em}}Real-time{{/em}}', { components: { em: <em /> } } ),
+	getButtonLabel: () =>
+		getJetpackCROActiveVersion() === 'v2'
+			? translate( 'Get Security {{em}}Real-time{{/em}}', { components: { em: <em /> } } )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Security {{em}}Real-time{{/em}}' )` *ES Score: 9*